com.unity.ml-agents/Runtime/DecisionRequester.cs

```diff
@@ -19,6 +20,26 @@ internal class DecisionRequester : MonoBehaviour
                  "of 5 means that the Agent will request a decision every 5 Academy steps.")]
         public int DecisionPeriod = 5;
 
+        /// <summary>
+        /// A delegate that can be used to customize the logic of when your Agent gets its
+        /// <see cref="Agent.RequestDecision"/> and <see cref="Agent.RequestAction"/> called.
+        /// If this value is not get, the default logic will use <see cref="DecisionPeriod"/> and
```

Comment:
```suggestion
        /// If this value is not set, the default logic will use <see cref="DecisionPeriod"/> and
```

Reply:
ah, I will move that sentence down to the actual member as well.  Thanks.